### ISSUE TITLE
Client side js file is requiring jimp

### ISSUE THREAD

**Issue by roderickhsiao** [akfish/node-vibrant#17]
We are using webpack for packing our client side js,

From the package.json https://github.com/akfish/node-vibrant/blob/master/package.json#L6 
`"browser": "lib/browser.js"`

and within the file 
https://github.com/akfish/node-vibrant/blob/master/lib/browser.js#L5 

we have 
`Vibrant.DefaultOpts.Image = require('./image').Browser;`

which `require('./image')`
has both server side and client side code.

even if we directly require https://github.com/akfish/node-vibrant/blob/master/lib/image/browser.js file,
we are still having `Image = require('./index');`
https://github.com/akfish/node-vibrant/blob/master/lib/image/browser.js#L5

which cause webpack error 

```js
ERROR in ./~/node-vibrant/~/jimp/index.js
Module not found: Error: Cannot resolve module 'fs' in /Users/rodhsiao/git/react-weather/node_modules/node-vibrant/node_modules/jimp
 @ ./~/node-vibrant/~/jimp/index.js 1:9-22

ERROR in ./~/node-vibrant/~/jimp/~/read-chunk/index.js
Module not found: Error: Cannot resolve module 'fs' in /Users/rodhsiao/git/react-weather/node_modules/node-vibrant/node_modules/jimp/node_modules/read-chunk
 @ ./~/node-vibrant/~/jimp/~/read-chunk/index.js 2:9-22

ERROR in ./~/node-vibrant/~/jimp/~/mime/mime.js
Module not found: Error: Cannot resolve module 'fs' in /Users/rodhsiao/git/react-weather/node_modules/node-vibrant/node_modules/jimp/node_modules/mime
 @ ./~/node-vibrant/~/jimp/~/mime/mime.js 2:9-22
```

**Comment by akfish** [comment]
Oops. I'll fix it soon.

**Comment by akfish** [comment]
Fixed in 2.0.1

**Comment by roderickhsiao** [comment]
Thanks for the prompt fix!
